src/Elm/Kernel/VirtualDom.js

```diff
@@ -1358,7 +1358,7 @@ function _VirtualDom_addDomNodesHelp(domNode, vNode, patches, i, low, high, even
 			}
 		}
 		// 4. this is needed for some edge cases
-		if(vKids[j].$ === __2_NODE && (childNodes[k].tagName || "").toLowerCase() !== vKids[j].__tag)
+		if(vKids[j].$ === __2_NODE && (childNodes[k].tagName || "").toLowerCase() !== vKids[j].__tag.toLowerCase())
```

Comment:
For consistency it might be better to do `(vKids[j].__tag || "").toLowerCase()`

Reply:
Thanks @rob-sokolowski !

So this was just a missed case in the jinjor patch where the virtual node could also have a non-lowercase name for some reason?

Given this case is already a fallback amongst fallbacks, this makes sense to me 👍  can't imagine why we'd have a null/undefined `__tag` but I guess it doesn't hurt to guard as well.

Reply:
@supermario updated, and also put added a bit of detail above. I also tested this change locally (branch that I used for testing, but don't plan to merge into my repo): https://github.com/project-fir/fir-sandbox/pull/38/files